### PR TITLE
Check forbidden chars for ids/attr names in URIs

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@kzangeli 

#1793

### CODE REVIEW

File: test/functionalTest/cases/1793_forbidden_id_chars_in_uri/1793_forbidden_id_chars_in_uri_attrs.test [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Forbidden chars for attribute names in URIs
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. PUT entity attribute with forbidden char in ID
```
Comment (kzangeli) [inline test/functionalTest/cases/1793_forbidden_id_chars_in_uri/1793_forbidden_id_chars_in_uri_attrs.test]: Copy 'n paste:   ID => ATTR ?

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -17,4 +17,4 @@
 - Add: support for geo:point type as a way of specifying location attribute in NGSIv2 (Issue #1038)
 - Add: type param for PUT entity in v2 (Issue #988, #992, #1000)
 - Fix: not detecting forbidden chars in entityID for PATCH v2 (Issue #1782)
-
+- Add: detect forbidden chars in ids and atrr names in URI (Issue #1793)
```
Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: type: atrr => attr

Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: ids -> entity ids (to distinguish among other possible ids, e.g. subscription ids)

Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed c2969ee

**Comment by fgalan** [comment]
LGTM (passing the ball to @kzangeli )

**Comment by kzangeli** [comment]
LGTM

File: test/functionalTest/cases/1793_forbidden_id_chars_in_uri/1793_forbidden_id_chars_in_uri_attrs.test (hunk 1 above)
Comment (crbrox) [inline test/functionalTest/cases/1793_forbidden_id_chars_in_uri/1793_forbidden_id_chars_in_uri_attrs.test]: Fixed e01a7cfcd46c8e7498daa981d9cdbc61ac3441f0

**Pull request merged**